### PR TITLE
Feature/accordion

### PR DESCRIPTION
**Pull request by gustaveen**
Redesigned brand pivot accordion

accordion

### CODE REVIEW

File: src/blocks/AccordionBlockBandPivot/AccordionBlockBandPivot.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+          <ExpandToggler onClick={toggleIsOpen}>
+            <AccordionTitleContent>{title}</AccordionTitleContent>
+            <ExpanderWrapper>
+              <ExpanderIcon isOpen={isOpen} />
+            </ExpanderWrapper>
+          </ExpandToggler>
+        </AccordionTitle>
+        <AnimateHeight height={isOpen ? 'auto' : 0}>
+          <AccordionContent
+            dangerouslySetInnerHTML={{ __html: paragraph && paragraph.html }}
```
Comment (palmenhq) [inline src/blocks/AccordionBlockBandPivot/AccordionBlockBandPivot.tsx]: `paragraph?.html`?

**Review by palmenhq — COMMENTED** [review]
nice!

File: src/blocks/AccordionBlockBandPivot/AccordionBlockBandPivot.tsx (hunk 1 above)
Comment (gustaveen) [reply src/blocks/AccordionBlockBandPivot/AccordionBlockBandPivot.tsx]: Of course!

**Review by palmenhq — APPROVED** [review]
ah, beautiful!

**Pull request merged**